RoadMap_SwiftUI/RoadMap_SwiftUI/ClimateView.swift

```diff
@@ -0,0 +1,318 @@
+                            colorPickerStackView
+                            stepperView
+                            ventStackView
+                        }
+                        .padding(.horizontal)
+                        Spacer()
+                    }
+                }
+            }
+        }
+        .navigationBarBackButtonHidden()
+    }
+    
+    // MARK: - Private properties
+    @Environment(\.presentationMode) var presentation
+    
+    @StateObject private var climateViewModel = ClimateViewModel()
+    
+    func selectColorToGradient() -> LinearGradient {
+        LinearGradient(colors: [climateViewModel.selectedColor, Color.bottomGradient], startPoint: .leading, endPoint: .trailing)
+    }
+    
+    private var actualTemperature: Int { Int(15.0 + climateViewModel.acCircleProgress) }
+    
+    private var colorPickerStackView: some View {
+        VStack(alignment: .center) {
+            ColorPicker("", selection: $climateViewModel.selectedColor)
+                .frame(width: 45)
+                .padding(.trailing, 10)
+            Text("Color")
+                .font(.subheadline)
+        }
+    }
+    
+    private var customAlertView: some View {
+        VStack {
+            if let url = URL(string: "https://www.tesla.com/support") {
+                Link(destination: url) {
+                    Image("teslaLogo")
+                        .resizable()
+                        .rotationEffect(Angle(degrees: 90))
+                        .foregroundColor(Color.lightShadow)
+                        .frame(width: 50, height: 50)
+                    Button("Tesla Support") {
+                        withAnimation(.linear) {
+                            climateViewModel.isAlertShown.toggle()
+                        }
+                    }
+                    .foregroundColor(.white)
+                }
+            }
+        }
+        .background(Rectangle()
+            .fill(Color.backgroundColor)
+            .frame(width: 200, height: 200)
+        )
+    }
+    
+    
+    private var ventStackView: some View {
+        VStack(alignment: .center) {
+            Image("carDoor")
+                .frame(width: 45)
+            Text("Vent")
+                .font(.subheadline)
+        }
+    }
+    
+    private var stepperView: some View {
+        CustomStepper(
+            value: Binding(
+                get: {
+                    Int(climateViewModel.acCircleProgress)
+                },
+                set: { newValue in
+                    if climateViewModel.range ~= newValue {
+                        climateViewModel.acCircleProgress = CGFloat(newValue)
+                    }
+                }
+            )
+        )
+        .frame(width: 80)
+    }
+    
+    private var acPowerStackView: some View {
+        HStack {
+            VStack(alignment: .leading) {
+                Text(climateViewModel.isPowerTapped ? "A/C is ON" : "A/C is OFF")
```

Comment:
```suggestion
                Text()
```
вынеси в модельку эту логику